app/models/user.rb

```diff
@@ -1,11 +1,16 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] You did a good job adding devise 🥇, however, make sure the `:confirmable` is enabled in the user model 👍. this is to make the application asks for confirmation of email. It'll be something like this:

```ruby
devise :database_authenticatable, :registerable,
		:recoverable, :rememberable, :validatable, :confirmable
``` 

Kindly, fix it 😃